pkg/flow/adapter/xslttransform/adapter_test.go

```diff
@@ -256,6 +257,62 @@ func TestXSLTTransformEvents(t *testing.T) {
 	}
 }
 
+func TestXSLTTransformKSINK(t *testing.T) {
+	testCases := map[string]struct {
+		xslt           string
+		inEvent        cloudevents.Event
+		expectedEvent  cloudevents.Event
+		expectCategory string
+	}{
+		"transform ok": {
+			xslt:           tXSLT,
+			inEvent:        newCloudEvent(tXML, cloudevents.ApplicationXML),
+			expectedEvent:  newCloudEvent(tOutXML, cloudevents.ApplicationXML),
+			expectCategory: tSuccessAttribute,
+		},
+	}
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			ceClient := adaptertest.NewTestClient()
+
+			env := &envAccessor{
+				EnvConfig: adapter.EnvConfig{
+					Component: tCloudEventSource,
+				},
+				BridgeIdentifier: tBridgeID,
+			}
+
+			ctx := context.Background()
+			style, err := parseXSLT(tc.xslt)
+			assert.NoError(t, err)
+
+			replier, err := targetce.New(env.Component, logtesting.TestLogger(t),
+				targetce.ReplierWithStatefulHeaders(env.BridgeIdentifier),
+				targetce.ReplierWithPayloadPolicy(targetce.PayloadPolicy(targetce.PayloadPolicyAlways)),
+				targetce.ReplierWithStaticResponseType(v1alpha1.EventTypeXSLTTransformError))
+			assert.NoError(t, err)
```

Comment:
Likewise, regarding `assert` vs `require`.
If you get an error here, we probably should interrupt the test immediately.

Reply:
Ive removed all of the replier logic. so this one is no longer there.